polyply/data/2016H66/polyvinyl_blocks.ff

```diff
@@ -312,3 +272,53 @@ PAM 3
 4 7 1 {"ifndef": "eq_polyply", "version": "1"}
 4 6 1 4.3755200e-04  1.0591530e-07 {"ifdef": "eq_polyply", "version": "2", "comment": "pairtype HC-O"}
 4 7 1 4.3755200e-04  1.0591530e-07 {"ifdef": "eq_polyply", "version": "2", "comment": "pairtype HC-O"}
+
+[ moleculetype ]
+PMAM 3
+[ atoms ]
+1  CH3 1 PMAM VC1 1  0.00 15.0350
+2  CH1 1 PMAM VC2 2  0.00 14.0270
+3  C   1 PMAM SC1 3  0.54 12.0110
+4  O   1 PMAM SC2 4 -0.54 15.9994
+5  N   1 PMAM SC3 5 -0.76 14.0067
+6  H   1 PMAM SC4 6  0.38 1.0080
+7  H   1 PMAM SC5 7  0.38 1.0080
+8  CH3 1 PMMA SB1 6  0.00 15.0350
+[ bonds ]
+1 2 2 gb_27 {"comment":"C,CHn-CHn,C"}
+2 3 2 gb_27 {"comment":"C,CHn-CHn,C"}
+3 4 2 gb_5 {"comment":"O-C"}
+3 5 2 gb_11 {"comment":"C-N,NZ,NE"}
+5 6 2 gb_2 {"comment":"N-H"}
+5 7 2 gb_2 {"comment":"N-H"}
+2 8 2 gb_27 {"comment":"C,CHn-CHn,C"}
+[ angles ]
+3 2 1 2 ga_15 {"comment":"CHn-CHn-C,CHn,OA,OE,NR,NT,NL"}
+; amide
+4 3 2 2 ga_30 {"comment":"CHn-C-O"}
+2 3 5 2 ga_19 {"comment":"CHn,C-C-OA,N,NT,NL"}
+4 3 5 2 ga_33 {"comment":"O-C-OA,N,NT,NL"}
+3 5 6 2 ga_23 {"comment":"H-NT,NZ,NE-C"}
+3 5 7 2 ga_23 {"comment":"H-NT,NZ,NE-C"}
+7 5 6 2 ga_24 {"comment":"H-NT,NZ-H"}
+;
+1 2 8 2 ga_15 {"comment":"CHn-CHn-C,CHn,OA,OE,NR,NT,NL"}
+8 2 3 2 ga_15 {"comment":"CHn-CHn-C,CHn,OA,OE,NR,NT,NL"}
+[ dihedrals ]
+1 2 3 4 1 gd_40 {"comment":"X-CHn-C,CR1-X"}
+2 3 5 6 1 gd_14 {"comment":"X-C-N,NT,NE,NZ,NR-X"}
+[ impropers ]
+3 4 5 2 2 gi_1 {"comment":"planar center"}
+5 3 6 7 2 gi_1 {"comment":"planar center"}
+[ pairs ]
+1 4 1
+1 5 1
+2 6 1
+2 7 1
+5 7 1
+8 4 1
+8 5 1
+4 6 1 {"ifndef": "eq_polyply", "version": "1"}
+4 7 1 {"ifndef": "eq_polyply", "version": "1"}
+4 6 1 4.3755200e-04  1.0591530e-07 {"ifdef": "eq_polyply", "version": "2", "comment": "pairtype HC-O"}
+4 7 1 4.3755200e-04  1.0591530e-07 {"ifdef": "eq_polyply", "version": "2", "comment": "pairtype HC-O"}
```

Comment:
The "ifndef version 1 vs version2" bit could be documented a bit more, i.e., having a couple of comment lines in this .ff file about this versions?
I'm not sure if we want to go for this, but I have the feeling that it could be useful.
In case, same for PAM above.